### DISCUSSION TITLE
Flatpak or AppImage installation

### DISCUSSION THREAD

**Discussion by rothgar** [streamdeck-linux-gui/streamdeck-linux-gui#71] [Ideas / Feature Requests]
Would it be possible to package the application with a portable installer? I imagine AppImage would be easier to access devices but flatpak probably has more adoption.

If someone PR'd a portable application version would you accept it?

Thanks

**Comment by coolapso** [comment]
Yes, it would be for sure accepted.

**Comment by docop** [comment]
Having an appimage can be really the best, 1 true portable file to run. No 2-3gig to download useless stuff as flatpack do and need to update every hrs.

**Reply by coolapso** [reply]
https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/88

Read the `Snap and Flatpacks` section thank you
